### PR TITLE
added image and description of API

### PR DESCRIPTION
**Pull request by aodhgan**

### CODE REVIEW

**Review by asselstine — CHANGES_REQUESTED** [review]
I would say the page you create could be moved into Concepts, and a reference page is added
That ref page should cover:

- What the API provides (claimable prizes)
- How to access the API (show actual URL format)
- What the data looks like (prize structure)

**Comment by asselstine** [comment]
Some things I noticed:

- The "Prizes API" section in Concepts isn't really a concept.  It's describing how the HTTP api is built.
- The references "Prizes API" section doesn't have a title "Prize API" with an introduction explaining what it does.  Don't rely on a link.

Changes I'd like to see:

- Change the "Prize API" concepts section to "Computing Prizes" and describe how to compute prizes **conceptually**.  I.e. the data needed and the formulas.  On that page you can then have sub-sections describing the tools (the CLI and the API which just wraps the CLI).
- The reference Prize API section is pretty good, but needs a proper title and in-line description of what it is.  Describe in one simple sentence what this does.
- The reference Prize API description should set the stage for the sub-sections below.  Each should have a tldr description.  [This page](http://localhost:3000/protocol/concepts/prize-pools-network) is a good example.  Description sets the stage for the more detailed content below.

**Comment by asselstine** [comment]
Couple more:

- change the word `degree` in the other sections to `tier`, as we should be consistent in our language
- don't use the word `awardable` prize.  it doesn't mean anything.

File: docs/reference/prize-api.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+```
+
+This data can then be imported into scripts for analysis using:
+
+```javascript
+// all winners data for draw 2 on mainet
+import allWinnersDraw15 from "./node_modules/v4PrizesData/api/prizes/1/0xb9a179dca5a7bf5f8b9e088437b3a85ebb495efe/draw/2/prizes";
+```
+
+# Prize Data Lifecycle Overview
```
Comment (aodhgan) [inline docs/reference/prize-api.md:153]: This section doesnt really fit here anymore. Just take the hit and delete it? @asselstine

**Comment by aodhgan** [comment]
Netlify builds dont seem to see static image files or something

Screen Shot 2021-12-13 at 11 19 48 AM

**Review by asselstine — COMMENTED** [review]
Broadly:

The API reference is pretty good.  But- you're right in that the Prize Data Lifecycle section isn't a good fit there.  I think that section should be moved to the v4-draw-results project README, because it explains how that pipeline works.

The "Computing Prizes" page doesn't actually explain how prizes are computed.  It talks about how the Draw Calculator contract computes the prizes and says that it's possible off-chain.  No links are provided, no details are provided.

Computing prizes should detail what the high-level steps of the algorithm are:

- Get the Draw & Prize Distribution from the contract DrawBuffer and PrizeDistBuffer
- compute the user's TWAB for the draw given the draw start, beacon period & offsets
- compute the number of picks a user gets
- for each pick, compute the PRNG and compare it to winning number

Then below that you can allude to how you can calculate it yourself using the lib or cli, or access the prizes we've computed using those tools via the API.

File: docs/concepts/computing-prizes.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+title: Computing Prizes
+sidebar_position: 4
+---
+
+# Computing Prizes
+
+This section outlines how the prizes are computed for each draw. The high-level steps of this process are:
+
+- Get the Draw & Prize Distribution from the DrawBuffer and PrizeDistributionBuffer contracts respectively
+- Compute the user's TWAB for the draw given the draw start teimstamp, beacon period & offsets
```
Comment (asselstine) [inline docs/concepts/computing-prizes.md:11]: *timestamp

**Review by asselstine — APPROVED** [review]

**Pull request merged**